tests/extendrtests/README.md

```diff
@@ -18,10 +18,10 @@ The Cargo.toml file hardcodes the relative path of the `extendr` libraries, thus
 in order to check this project locally you need to run:
 
 ```r
-devtools::check(check_dir = "check")
+rcmdcheck::check(check_dir = "check")
```

Comment:
I am afraid it should be `rcmdcheck::rcmdcheck`
https://github.com/extendr/extendr/blob/702b24ba9e871010779cf28906fedffc3b34802a/.github/workflows/test.yml#L177

Reply:
ahh right! missed that! Should be fixed now!